src/clarity-angular/data/datagrid/render/dom-adapter.spec.ts

```diff
@@ -3,6 +3,8 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
+import {ɵPLATFORM_BROWSER_ID} from "@angular/common";  // Might be hacky but it is exported.
```

Comment:
Typo `ɵPLATFORM_BROWSER_ID` ?

Reply:
Nope, that is what it is. Its similar to how we handle the focus trap export. Its public, but not overly public.

Reply:
Did you look into maybe using `platformBrowserTesting()` or `platformBrowser()` to create the platform and get the ID from there? Yes, it's much heavier, but it would use 100% public API, which is safer.
Unless you feel confident this `ɵPLATFORM_BROWSER_ID` will never be changed in a minor version, which sounds a bit optimistic. 😛

Reply:
It is a public API, so it should go through a deprecation before change. However it must be excluded from the docs for some reason. I guess there are no guarantees in life though

I'll look at the testing options, but no documentation on it anywhere...

Reply:
No, it's actually in the public API for internal reasons, and is not actually public. As such, they reserve the right to change this without even mentioning it in the changelog. They have several exports like that, all of them being here to allow their own packages to communicate.
I remember they did change the same ones for animations between 2 minors, with no warning and no note about it.

Reply:
Well I've already refactored for the noop dom adapter and this won't be necessary then so 👍